archunit/src/test/java/com/tngtech/archunit/lang/ArchRuleTest.java

```diff
@@ -159,6 +161,20 @@ public void finish(ConditionEvents events) {
         assertThat(condition.eventsFromFinish.getViolating()).hasSize(1);
     }
 
+    @Test
+    public void empty_input_throws_an_assertion_exception_by_default() {
```

Comment:
It's an `assertion_**error**` :wink: Also, given the other comments probably "rule_that_fails_to_check_any_class..."? :thinking:

Reply:
I renamed the test methods, but I'm not sure if they are good enough. Maybe you have better suggestions?😄